Sources/XCMetricsBackendLib/Statistics/Controllers/StatisticsController.swift

```diff
@@ -58,10 +98,8 @@ public struct StatisticsController: RouteCollection {
     /// }
     /// ```
     public func buildStatus(req: Request) throws -> EventLoopFuture<Page<BuildStatusResult>> {
-        return Build.query(on: req.db)
-            .field(\.$id)
-            .field(\.$buildStatus)
-            .paginate(for: req)
-            .map { $0.map { BuildStatusResult(id: $0.id!, buildStatus: $0.buildStatus) } }
+        let page = Int(req.query["page"] ?? "") ?? 1
```

Comment:
I think this can be a bit more readable (the same in the line above):
```suggestion
        let page = Int(req.query["page"] ?? "1") 
```